staging/src/k8s.io/pod-security-admission/policy/registry.go

```diff
@@ -69,6 +70,9 @@ func (r *checkRegistry) EvaluatePod(lv api.LevelVersion, podMetadata *metav1.Obj
 		results = append(results, check(podMetadata, podSpec))
 	}
 	if lv.Level == api.LevelBaseline {
+		for _, check := range r.overlappingBaselineChecks[lv.Version] {
```

Comment:
marking checks as overlapping and skipping them without positive verification that the specific check they overlap with is also getting run makes me slightly nervous

the integration test failures are related, and make it seem like some expected checks aren't getting run